hammer/formal/conformal/__init__.py

```diff
@@ -32,8 +32,10 @@ def env_vars(self) -> Dict[str, str]:
     @property
     def start_cmd(self) -> List[str]:
         """ Generate required startup command based on the requested check and license level """
-        lec_bin = self.get_setting("formal.conformal.conformal_lec_bin")
-        ccd_bin = self.get_setting("formal.conformal.conformal_ccd_bin")
+        lec_bin = self.get_setting("formal.conformal.conformal_lec_bin") + \
+                  self.get_setting("formal.conformal.conformal_lec_bin_args")
```

Comment:
This doesn't match the key in the `defaults.yml` below.

I think for the rest of them I'd prefer doing `<tool>_extra_args` for clarity instead of `<tool>_bin_args` or `<tool>_args`